api/types/parser.go

```diff
@@ -223,45 +223,6 @@ func (ctx *Context) GetIdentifier(fields []string) (interface{}, error) {
 	}
 }
 
-// NewParserFn returns function that creates parser of 'where' section
```

Comment:
Sorry for putting this out of context - can you also fix the typo in RuleContext.GetResource:
unpecified -> unspecified
?